modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js

```diff
@@ -1073,6 +1074,30 @@ return view.extend({
 					    bssid = ss.children[5],
 					    encr;
 
+					/* 802.11v settings start */
+					// Probe 802.11v support (needs full hostapd/wpad) via EAP support (full hostapd has EAP)
+					if(L.hasSystemFeature('hostapd', 'eap'))
```

Comment:
Please leave a space between control statement keywords and the condition (e.g. `if (...)`, `for (...)`, `while (...)`, `switch (...)`)

Reply:
OK.